src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
```

Comment:
I get that, but I'm also very wary of creating a big grab bag file that has tons of unrelated structs. Id rather keep them together with the code that needs them and have sys call.h include multiple files than have every file than needs any struct including ALL of them unnecessarily.